pallets/proxy-rmrk-core/src/misc.rs

```diff
@@ -14,9 +14,13 @@
 // You should have received a copy of the GNU General Public License
 // along with Unique Network. If not, see <http://www.gnu.org/licenses/>.
 
+//! Miscellaneous helpers and utilities used by the proxy pallet.
+
 use super::*;
 use codec::{Encode, Decode, Error};
 
+/// Match errors from one type to another and return an error
+/// if a match is successful.
```

Comment:
Can you specify what error will be returned?

Reply:
The errors are provided to the macro, it's a bit complicated :) Take a look [here](https://github.com/UniqueNetwork/unique-chain/blob/doc/rmrk/pallets/proxy-rmrk-core/src/lib.rs#L2072), the macro is a utility and accepts any pattern matching mapper, despite its name.